docs/src/man/reading.md

```diff
@@ -164,16 +168,17 @@ tcagTTAAGATGGGAT
 
 To read a file containing such records, one could use:
 ```julia
-reader = open(FASTQReader, "reads.fastq", Seq.SANGER_QUAL_ENCODING)
-seqrec = eltype(reader)()
-while !eof(reader)
-    read!(reader, seqrec)
-    # ... process sequence
+reader = open(FASTQReader, "reads.fastq")
```

Comment:
Would prefer to keep the more performant in-place parser style here. Also, please make clear in the docs that the default qual encoding is `:sanger`